src/sass/textbox/textbox.scss

```diff
@@ -244,4 +244,8 @@ textarea.textbox__control[readonly]:focus {
 
 .textbox--fluid {
     width: 100%;
+
+    & .textbox__control {
```

Comment:
Typically in `skin` we avoid nesting whenever possible because it helps with scanability and searchability, would you mind separating this out into a separate selector?
```
.textbox--fluid .textbox__control {
```

Reply:
Thanks, done